src/Build.UnitTests/Graph/ProjectGraph_Tests.cs

```diff
@@ -64,6 +64,21 @@ public void ConstructWithSingleNode()
             }
         }
 
+        [Fact]
```

Comment:
I'm a little confused—what does this test have to do with the rest of the PR?

Reply:
Asserts that metrics are actually computed.

Reply:
That still feels perpendicular to this PR, since it isn't about computing anything but rather logging how long it took.

Reply:
I don't understand your concern @Forgind. This PR introduces a new public field, and this is a lightweight test that the field is populated.